ui/assets/extensions/intercom-widget-extension.js

```diff
@@ -6,7 +6,7 @@
    */
   angular
     .module('openshiftOnlineConsoleIntercomExtension', ['openshiftConsole'])
-    .run(function($rootScope, $q, HawtioExtension) {
+    .run(function($rootScope, $q, extensionRegistry) {
```

Comment:
You should just be able to remove HawtioExtension since it wasn't used anywhere. extensionRegistry is not needed.

Reply:
@spadgett thanks, and we're still using hawtioPluginLoader right?

Reply:
You can use `pluginLoader` or `hawtioPluginLoader` in 3.10, but it has to be `hawtioPluginLoader` in earlier versions.